Cocktail/Cocktail/ViewControllers/CocktailDetailViewController.swift

```diff
@@ -171,8 +171,25 @@ class CocktailDetailViewController: UIViewController {
         recipeGuideLabel.text = "Recipe".localized
         myTipGuideLabel.text = "Tip".localized
         ingredientsGuideLabel.text = "Ingredients".localized
+        
+        likeButton.addAction(UIAction(handler: { [weak self]_ in
+            guard let self = self, let cocktailData = self.cocktailData else { return }
```

Comment:
```suggestion
            guard let self = self, 
            let cocktailData = self.cocktailData else { return }
```
guard 문에 복수의 optional 값을 처리할 때 구분해 주는 것이 가독성 측면에서 나음